### ISSUE TITLE
@puppeteer/browsers package doesn't install dependencies

### ISSUE THREAD

**Issue by SamStephens** [GoogleChromeLabs/chrome-for-testing#121]
The [README.md](https://github.com/GoogleChromeLabs/chrome-for-testing/README.md) and [official blog post](https://developer.chrome.com/blog/chrome-for-testing/) both make it look as though installing via `@puppeteer/browsers` will give you working Chrome browsers.

However, installing this way does not provide all the required dependencies. If I run my reproduction Dockerfile

```
FROM python:3.11-bookworm # Based on Debian Bookwork

RUN apt-get update --fix-missing && apt-get upgrade -y && apt-get dist-upgrade -y

RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
RUN apt-get install -y nodejs

RUN npx @puppeteer/browsers install chrome@123.0.6312.58 | \
  awk '{print $2}' | \
  xargs -I {} ln --symbolic {} /usr/local/bin/chrome
RUN npx @puppeteer/browsers install chromedriver@123.0.6312.58  | \
  awk '{print $2}' | \
  xargs -I {} ln --symbolic {} /usr/local/bin/chromedriver

ENTRYPOINT /usr/local/bin/chromedriver
```

I get the error

```
/usr/local/bin/chromedriver: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory
```

This is talked about somewhat in https://github.com/GoogleChromeLabs/chrome-for-testing/issues/55, where people detail their various workarounds for installing the dependencies. I've raised this issue separately, because https://github.com/GoogleChromeLabs/chrome-for-testing/issues/55 is talking about instructions being missing entirely for installation from the ZIP files, whereas my concern is that the provided instructions for using `@puppeteer/browsers` are incomplete and misleading.

There should be an officially blessed way to install the dependencies; it's unfortunate that the consumers of chrome-for-testing are guessing how to best install the dependencies.

**Comment by thiagowfx** [comment]
Proposed workaround: https://serverfault.com/questions/577942/install-only-dependencies-of-a-given-package-in-debian-or-ubuntu-apt

TL;DR:

```
sudo apt `apt-cache depends google-chrome | awk '/Depends:/{print$2}'`
```

Nonetheless, I'd argue that this bug belongs to https://github.com/puppeteer/puppeteer instead of this repository.

**Comment by SamStephens** [comment]
@thiagowfx I think it probably belongs here, because users who download the zip files themselves without using Puppeteer also face the same issue.

I'm aware of the various workarounds I've seen being discussed.

My point here is that Chrome for Testing should provide **official guidance**; that it is not "a dedicated flavor of Chrome targeting the testing use case" [1] if the repository does not provide a self-contained way to install fully functioning Chrome (which includes dependencies).

[1] From https://developer.chrome.com/blog/chrome-for-testing/

**Comment by ngan** [comment]
FWIW, this was the trick I used (from Dockerfile, on Ubuntu):

```
    wget --quiet -O - https://dl.google.com/linux/linux_signing_key.pub | apt-key add - \
    && echo "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list \
    && apt-get update \
    && apt-get install --yes --no-install-recommends google-chrome-stable \
    && apt-get remove --yes google-chrome-stable
```

Essentially install `google-chrome-stable` and its dependencies. Then uninstall `google-chrome-stable` leaving its dependencies behind.

**Comment by SamStephens** [comment]
@ngan please, no more workarounds. I've seen them all. That's not the point. This is an product of Google, one of the largest tech companies in the world. We shouldn't need workarounds. There should be **official guidance**.

**Comment by zdm** [comment]
It would be nice if you will put simple bash script, which will install deps, inside each zip archive.
